packages/eas-cli/src/credentials/android/api/GraphqlClient.ts

```diff
@@ -164,14 +174,6 @@ export async function createAndroidAppBuildCredentialsAsync(
       graphqlClient,
       appLookupParams
     );
-  const buildCredentialsList = androidAppCredentials.androidAppBuildCredentialsList;
```

Comment:
If we are creating new credentials and that specific app did not have any before then the new set should be set to default automatically (without prompting the user if they want to set it to default). I'm not sure if it's event valid state to not have a default credentials.

Make sure to test that on a new Expo app.